app/src/main/res/drawable/ic_overview_white_24dp.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This file seems redundant.
Check ic_overview_white_24dp

Also, there was one comment on #230 which was not addressed and actually it was first resolved without making any change and then I marked that as unresolved but there was no reply on that.
#230 (comment)

Basically, all tabs icons names should end with "_24dp.xml" or "_24.xml" but do not use both. I would prefer "_24dp.xml" considering that android studio uses this convention by default.

Reply:
Removed ic_overview_white_24dp

Reply:
@nikitamarysolomanpvt This is weird. 

**Also, there was one comment on #230 which was not addressed and actually it was first resolved without making any change and then I marked that as unresolved but there was no reply on that.
#230 (comment)

Basically, all tabs icons names should end with "_24dp.xml" or "_24.xml" but do not use both. I would prefer "_24dp.xml" considering that android studio uses this convention by default.**

I am writing this comment for the fourth time, twice in previous PR and twice in this PR.

1. Always read the entire comment.
2. Click "Resolve" only once you have solved or replied to that entire comment.

Reply:
Done